src/map/packets/zone_in.h

```diff
@@ -27,11 +27,12 @@
 #include "basic.h"
 
 class CCharEntity;
+struct EventInfo;
 
 class CZoneInPacket : public CBasicPacket
 {
 public:
-    CZoneInPacket(CCharEntity* PChar, int16);
-};
+    CZoneInPacket(CCharEntity* PChar, const EventInfo*);
+    };
```

Comment:
This ending is missaligned, i think

Reply:
Thanks, was transfering the bulk of the code our of ASB into LSB